source/api/commands/as.md

```diff
@@ -46,22 +45,44 @@ The name of the alias to be referenced later within a {% url `cy.get()` get %} o
 
 ## DOM Element
 
+Aliasing a DOM element and then {% url getting get %} that aliased element.
+
 ```javascript
-cy.route('PUT', /^\/users\/\d+/, 'fixture:user').as('userPut')
-cy.get('form').submit()
-cy.wait('@userPut')
-  .its('url').should('contain', 'users')
+it('disables on click', function () {
+  cy.get('button[type=submit]').as('submitBtn')
+  cy.get('@submitBtn').click().should('be.disabled')
+})
```

Comment:
The original example wasn't actually aliasing a DOM element 😕so I changed it to one that does actually alias a DOM element.